content/04.network_applications.md

```diff
@@ -14,9 +14,32 @@
 
 #### Deep Learning
 
-1. Define node neighborhoods
-2. Talk about random walks 
-3. Talk about auto encoders random walk independent approaches 
+Deep learning is a paradigm that excels in mapping high dimensional data into a low dimensional space.
+Most techniques under this paradigm have flourished based on a popular natural language processing approach called Word2Vec [@arxiv:1310.4546; @arxiv:1301.3781].
+Word2Vec projects words into a low dimensional space that preserves their semantic meaning.
```

Comment:
```suggestion
Word2Vec projects words into a low dimensional space that preserves their semantic meaning.
```
```suggestion
The goal of a Word2Vec model is to project words into a low dimensional space that preserves their semantic meaning.
```
What is caps supposed to be like here on Word2Vec? I feel like I usually see it as word2vec but I'm not sure if that's the canonical representation.